src/uu/cut/src/cut.rs

```diff
@@ -337,6 +347,88 @@ fn cut_files(mut filenames: Vec<String>, mode: &Mode) {
     }
 }
 
+// This is temporary helper function to convert OsString to &[u8] for unix targets only
+// TODO Remove this function and re-implement the functionality in each place that calls it
+// for all targets using https://doc.rust-lang.org/nightly/std/ffi/struct.OsStr.html#method.as_encoded_bytes
+// once project's MSRV is bumped up to 1.74.0+ so that function becomes available
+// For now - support unix targets only and on non-unix (i.e. Windows) will just return an error if delimiter value is not UTF-8
+fn os_string_as_bytes(os_string: &OsString) -> UResult<&[u8]> {
```

Comment:
You can use `&OsStr` for this (and also get access to `'static` lifetimes that way).